iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj

```diff
@@ -30,10 +30,10 @@
     <PackageReference Include="MSTest.TestFramework" Version="2.0.0" />
     <PackageReference Include="NSubstitute" Version="3.1.0" />
     <PackageReference Include="Moq" Version="4.10.1" />
+    <Reference Include="System.Web" />
```

Comment:
I would like to know the error thrown if you don't specify global import for System.Web

Reply:
Error	CS0234	The type or namespace name 'HttpUtility' does not exist in the namespace 'System.Web' (are you missing an assembly reference?)	Microsoft.Azure.Devices.Client.Tests (netcoreapp3.1)	C:\src\azure-iot-sdk-csharp\iothub\device\tests\Mqtt\MqttIotHubAdapterTest.cs	132	N/A